django/http/multipartparser.py

```diff
@@ -302,8 +328,13 @@ def handle_file_complete(self, old_field_name, counters):
             if file_obj:
                 # If it returns a file object, then set the files dict.
                 self._files.appendlist(
```

Comment:
Is `self._files.appendlist(force_text(old_field_name, self._encoding, errors='replace'), file_obj)` too long?